autospec/license.py

```diff
@@ -76,31 +76,38 @@ def add_license(lic):
     return result
 
 
+def decode_license(license):
+    def try_with_charset(license, charset):
+        if not charset:
+            return
+
+        try:
+            return data.decode(license, charset)
+        except UnicodeDecodeError:
+            if license_charset in ('ISO-8859-1', 'ISO-8859-15'):
+                if b'\xff' in data:
+                    return try_with_charset(data, 'ISO-8859-13')
+                if b'\xd2' in data and b'\xd3' in data:
+                    return try_with_charset(data, 'mac_roman')
```

Comment:
`data` is not defined in this function... I presume the correct variable name is `license`?

Reply:
Oops.